### DISCUSSION TITLE
Displaying 3 Items with WrapAround

### DISCUSSION THREAD

**Discussion by florianbruening** [ismail9k/vue3-carousel#151] [Q&A]
Hello there,

im trying to display 3 items with wrap around. 

My issue right now:

The right most item is not displayed and if i navigate via the buttons or the pagination it always scrolls left.
```
<carousel
      wrapAround
      :items-to-show="2"
      snap-align="center"
    >
      <slide v-for="num in 3" :key="num">
        <div class="h-100 w-full bg-red-500">
          Test {{ num }}
        </div>
      </slide>
      <template #addons="{ slidesCount }">
        <Navigation v-if="slidesCount > 1" />
        <pagination v-if="slidesCount > 1" />
      </template>
</carousel>
```
Having 3 items it looks like this:
![image](https://user-images.githubusercontent.com/69163412/153229307-33f4baeb-2fb5-4e09-8de1-776fd45a7eed.png)

As soon as i display 4 items it works as expected:
![image](https://user-images.githubusercontent.com/69163412/153229436-50b55a46-879e-4da2-9c32-b1c7ef2457b8.png)

Is there something i can do to have the last behaviour even if i only have 3 items?

**Comment by grantholle** [comment]
What I've done is doubled my items, but I'm not using the navigation component (just arrows).
